modules/hosted-che/partials/ref_about-try-in-web-ide-github-action.adoc

```diff
@@ -0,0 +1,18 @@
+// Module included in the following assemblies:
+//
+// hosted-che
+
+
+[id="about-try-om-web-ide-github-action_{context}"]
+
+The link:https://github.com/marketplace/actions/try-in-web-ide[Try in Web IDE GitHub action] can be added to a GitHub repository workflow to help reviewers quickly test pull requests on Eclipse Che hosted by Red Hat.
+The action achieves this by listening to pull request events and providing provides a link:https://www.eclipse.org/che/docs/che-7/end-user-guide/creating-a-workspace-from-remote-devfile/#creating-a-workspace-from-the-default-branch-of-a-git-repository_che[factory URL] by creating a comment, a status check, or both.
+This factory URL creates a new workspace (link:https://www.eclipse.org/che/docs/che-7/end-user-guide/creating-a-workspace-from-remote-devfile/#creating-a-workspace-from-a-feature-branch-of-a-git-repository_che[from the pull request branch]) on Eclipse Che hosted by Red Hat.
+
+NOTE: The Che documentation repository (https://github.com/eclipse/che-docs) is a real-life example where the Try in Web IDE GitHub action helps reviewers quickly test pull requests. Experience the workflow by navigating to a recent pull request and opening a factory URL.
```

Comment:
Wanted to mention, this would be true if https://github.com/eclipse/che-docs/pull/2113 is merged